src/app/college/page.js

```diff
@@ -1,23 +1,17 @@
+//useSearchParams only works in Client Components.So, Add we need  "use client" directive at the top of the file to use it
```

Comment:
Please change the comment to the following:

The "use client" is used to declare a boundary between a Server and Client Component modules. This means  that by defining a "use client" in a file, all other modules imported into it, including child components, are considered part of the client bundle - and will be rendered by React on the client. In this file useSearchParams is a client component. Read more: https://nextjs.org/docs/app/building-your-application/rendering/client-components#using-client-components-in-nextjs